content/docs/12.5-vertex/60-system-management/10-tenant-management.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Tenant Management"
+metaTitle: "Tenant Management"
+metaDescription: "Learn how to create and remove tenants in Palette VerteX."
+icon: ""
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+# Overview
+
+Tenants are isolated environments in Palette VerteX that contain their own clusters, users, and resources. You can create multiple tenants in Palette VerteX to support multiple teams or projects. Instructions for creating and removing tenants are provided below.
+
+
+<br />
+
+# Create a Tenant
+
+You can create a tenant in Palette VerteX by following these steps.
+
+
+## Prerequisites
+
+* Access to the Palette VerteX system console.
+
+
+## Enablement
+
+1. Log in to the Palette VerteX system console.
+
+
+2. Navigate to the left **Main Menu** and select **Tenant Management**.
+
+
+3. Click **Create New Tenant**.
+
+
+4. Fill out the **Org Name** and the properties of the admin user by providing the **First Name**, **Last Name**, **Email**.
+
+
+5. Confirm your changes.
+
+
+6. From the tenant list view, find your newly created tenant and click on the **three dots Menu**. Select **Activate** to activate the tenant.
+
+  <br />
+
+  ![View of a tenant activation option](/vertex_system-management_tenant-management_activate-tenant.png)
+
+7. A popup box will present you with an activation URL. Copy the URL and paste it into your browser to activate the tenant.
```

Comment:
```suggestion
7. A pop-up box will present you with an activation URL. Copy the URL and paste it into your browser to activate the tenant.
```

Reply:
I know Grammarly doesn't snag this, but it seems "pop-up" should be hyphenated because it modifies "box".

Reply:
Thank you @ritawatson